src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -1,6 +1,34 @@
 package com.sap.piper
 
 class ConfigurationHelper implements Serializable {
+    static class ConfigLoader implements Serializable {
```

Comment:
Do we need this new class? I think the methods could also be part of the class ConfigurationHelper. 
Furthermore, we would introduce two classes with similar names (ConfigurationLoader and ConfigLoader)

Reply:
I would need to change the class properties definition (remove `final`) to do this.

Reply:
Okay. Why shouldn't we do that?